SConstruct

```diff
@@ -345,6 +341,8 @@ if not env.GetOption('clean'):
 	boost_versions = ['-1_84'] # This is a bit of a hack. :(
 	suffixes = ['-mt', f'-mt-x{env["bits"]}']
 
+	zlib = 'zlib' if (platform == "win32" and 'mingw' not in env["TOOLS"]) else 'z'
+	check_lib(zlib, 'zlib', [], [])
```

Comment:
I'm confused at how this makes it be bundled…? And it's not making it be bundled on _all_ platforms, is it?